Sources/MongoSwift/ChangeStreamEvent.swift

```diff
@@ -145,6 +148,16 @@ public struct ChangeStreamEvent<T: Codable>: Codable {
             self.ns = ns
         }
 
+        // `to` only exists in `rename` events so similar implementation as `ns`
```

Comment:
Makes sense. I just collapsed the `to` initialization all into one `if (rename) { _ } else { _ }` instead of doing the `do`/`guard`/`catch` implementation of `ns`.